hack/build.sh

```diff
@@ -36,11 +40,19 @@ fi
 
 export CGO_ENABLED=0
 MODE="${MODE:-release}"
-# build terraform binaries before setting environment variables since it messes up make
-make -C terraform all
 
-# Copy terraform parts to embedded mirror.
-copy_terraform_to_mirror
+# Build terraform binaries before setting environment variables since it messes up make
+if test "${SKIP_TERRAFORM}" != y
+then
+  make -j8 -C terraform all
```

Comment:
It's not clear to me that using the `-j` jobs flag is really enabling the jobs to run in parallel, as the manpages say:
```shell
$ make --help | rg "Allow N jobs"
  -j [N], --jobs[=N]          Allow N jobs at once; infinite jobs with no arg.
```

Reply:
Make by defaults run every job/target in sequential order, this allows Make to run 8 jobs at once, it still respects dependencies though

Reply:
In other words, it'll run N jobs in parallel, if they can run in parallel, which our Makefile for both terraform and CAPI allow

Reply:
> Allow N jobs at once; infinite jobs with no arg.

Sounds like `Make` will automatically try to parallelize as many jobs as possible?

Reply:
Yes but it's defaulted to 1, so there is effectively no parallelization

Reply:
The manpages make it sound like the default is ∞